src/client/src/rt-styleguide/components/Button.tsx

```diff
@@ -111,7 +111,7 @@ export class ButtonGroup extends React.Component<ButtonStyleProps> {
   }
 }
 
-const BaseElement = styled.button<ButtonStyleProps>`
+const StyledBase = withProps({ role: 'button' })(styled.div`
```

Comment:
Are we adding `recompose` back just for this one case?

Reply:
I think the right framing is with consideration that `Emotion` is an  incomplete implementation of `styled-components`. While `styled-components` implements the `attrs` method to address this very common use-case, the author of `Emotion` refuses to — instead they prescribe this approach. As the officially documented approach to an existing and expected API I think we should resist introducing additional indirection or variation on a variation.

I will look to see if I can remove it simply by adding it to the wrapping components — as this component is not exported. However, where similar cases emerge I believe this framing is the correct approach to such an issue.